docs/Dashboard/user_guide.md

```diff
@@ -1 +1,23 @@
-# Pbench Dashboard User Guide
+# Pbench Dashboard 
+
+Pbench Dashboard is the web-based platform for consuming indexed performance benchmark data. It provides data curation capabilities for the performance datasets.
+
+The landing page is the browsing page where the user can view the list of public datasets. Those datasets can be filtered based on name and/or uploaded time.
+
+![Browsing Page](assets/images/BrowsingPage.png)
+
+On logging in, the user can view the Overview Page which is the data curation page.
```

Comment:
Maybe mention the "Login" button at the top right? I think that when looking at the UI it's fairly obvious, but it wouldn't hurt to mention it. I'm not sure whether it makes sense to mention that this will redirect to a Keycloak OIDC instance...